languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -55658,6 +55658,24 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Zeichensetzung -->
     <!-- ====================================================================== -->
     <category id="PUNCTUATION" name="Zeichensetzung">
+        <rule id="KOMMA_OHNE_DASS" name="Komma vor 'ohne dass'" default="temp_off">
+            <antipattern>
+                <token postag="KON:NEB" postag_regexp="yes"/>
+                <token>ohne</token>
+                <token>dass</token>
+            </antipattern>
+            <pattern>
+                <token postag="VER.*|SUB.*|PRO:PER.*" postag_regexp="yes"/>
+                <marker>
+                    <token>ohne</token>
+                    <token>dass</token>
+                </marker>
+            </pattern>
+            <message>Der Konnektor 'ohne dass' leitet einen Nebensatz ein, der durch ein Komma getrennt werden muss.</message>
+            <suggestion>, ohne dass</suggestion>
+            <example correction=", ohne dass">Ich bin in die Schule gegangen<marker> ohne dass</marker> ich meine Hausaufgaben machen konnte.</example>
+            <example>Ich sage es ihr, ohne dass ich mich schlecht fühlen muss und ohne dass sie es falsch aufnimmt..</example>
```

Comment:
Zwei Punkte am Ende

... und die Sache mit SUBJUNKTION_KOMMA[1] (siehe Kommentar in _Conversation_)

Reply:
Schaue, ob ich die beiden Regeln 'ohne dass' und SUBJUNKTION_KOMMA[1] evtl. zusammennehmen kann.

Reply:
Ich habe die Regel 'SUBJUNKTION_KOMMA[1]' ergänzt.
Mit dieser Regel 'KOMMA_OHNE_DASS' als Subrule für 'SUBJUNKTION_KOMMA[1]' sollte es funktionieren.

Reply:
Cool! Machst du diese Regel dann noch zur Subrule, wie von dir erwähnt?

Reply:
Ist jetzt eine Subrule und alles könnte gemergt werden.

Reply:
Die Regel erscheint bei mir immer noch als eigenständige Regel und nicht als Subrule…